packages/react-next/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -491,6 +498,11 @@ export const CalloutContentBase = React.memo(
       return null;
     }
 
+    // We probably just switched focus context to another window. Don't rerender
+    if (!targetWindowRef.current.document.hasFocus()) {
```

Comment:
I think that it might be unsafe to conditionalize render based on this. The event listeners, refs, and other parts of the menu that interact directly with the dom might not get cleaned up properly. If there is something going with focus returning, you can use the new onFocusRestore to prevent focus from returning in whatever scenario you might want.

Reply:
I see no such prop in file / repo. Did you mean `shouldRestoreFocus` prop available?

Reply:
Hey @jodom! Yeah, I think @joschect referred to `shouldRestoreFocus`.

Reply:
Oh yeah, it's actually `onFocusRestore` prop on Callouts :) . Thanks